manifests/server/grant.pp

```diff
@@ -103,8 +138,9 @@
   }
 
   $_unless = $unless_function ? {
-      false   => undef,
-      default => "SELECT 1 WHERE ${unless_function}('${role}',
+      false    => undef,
+      'custom' => $custom_unless,
+      default  => "SELECT 1 WHERE ${unless_function}('${role}',
```

Comment:
The `is_pe` bit is caused by recent changes in puppetlabs-concat. The fix is at https://github.com/puppetlabs/puppetlabs-concat/pull/270